### PR TITLE
[keyboard] add bear_face/v2

### PR DESCRIPTION
**Pull request by chemicalwill**
## Description

* restructured keyboard dir for multiple versions
* added keymap default_iso.c
* fixed .json layouts for ISO
* reformatted hard tabs to soft tabs
* updated readmes

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* none

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by chemicalwill** [comment]
@zvecr thanks. Could you elaborate on what targeting `develop` would entail?

File: keyboards/bear_face/rules.mk [hunk 1]
```diff
@@ -2,20 +2,12 @@
 MCU = atmega32u4
```
Comment (noroadsleft) [inline keyboards/bear_face/rules.mk:2]: Please add `DEFAULT_FOLDER = bear_face/v1` to the end of this file.

Comment (chemicalwill) [reply keyboards/bear_face/rules.mk:2]: fixed

File: keyboards/bear_face/config.h [hunk 2]
```diff
@@ -20,9 +20,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xCEEB
+#define VENDOR_ID       0xFEED
 #define PRODUCT_ID      0x09f5
-#define DEVICE_VER      0x0001
 #define MANUFACTURER    chemicalwill
 #define PRODUCT         bear_face
 #define DESCRIPTION     Vortex Race 3 programmable PCB replacement
```
Comment (fauxpark) [inline keyboards/bear_face/config.h:27]:
```suggestion
```

File: keyboards/bear_face/v1/info.json [hunk 3]
```diff
@@ -0,0 +1,102 @@
+{
+    "keyboard_name": "bear_face",
+    "url": "https://github.com/chemicalwill/bear_face_pcb",
+    "maintainer": "chemicalwill",
+    "width": 15,
+    "height": 6,
+    "layouts": {
+        "LAYOUT_83_ansi": {
+            "key_count": 83,
```
Comment (fauxpark) [inline keyboards/bear_face/v1/info.json:9]:
```suggestion
```

File: keyboards/bear_face/v2/info.json [hunk 4]
```diff
@@ -0,0 +1,197 @@
+{
+    "keyboard_name": "bear_face",
+    "url": "https://github.com/chemicalwill/bear_face_pcb",
+    "maintainer": "chemicalwill",
+    "width": 15,
+    "height": 6,
+    "layouts": {
+        "LAYOUT_83_ansi": {
+            "key_count": 83,
```
Comment (fauxpark) [inline keyboards/bear_face/v2/info.json:9]:
```suggestion
```

File: keyboards/bear_face/v2/info.json [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+                {"label":"Fn", "x":11, "y":5},
+                {"label":"Ctrl", "x":12, "y":5},
+                {"label":"Left", "x":13, "y":5},
+                {"label":"Down", "x":14, "y":5},
+                {"label":"Right", "x":15, "y":5}
+            ]
+        },
+
+        "LAYOUT_84_iso": {
+            "key_count": 84,
```
Comment (fauxpark) [inline keyboards/bear_face/v2/info.json:103]:
```suggestion
```

File: keyboards/bear_face/rules.mk [hunk 6]
```diff
@@ -29,3 +21,5 @@ MIDI_ENABLE = no            # MIDI controls
 UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
 AUDIO_ENABLE = no           # Audio output on port C6
```
Comment (fauxpark) [inline keyboards/bear_face/rules.mk:23]:
```suggestion
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output6
```
And could you remove the `MIDI_ENABLE` line above this as well? (GitHub doesn't let me extend the suggestion).

**Comment by chemicalwill** [comment]
@drashna for whatever reason, `qmk cformat` isn't catching some of my tabs, but they should be fixed now

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**